src/ui/marker.js

```diff
@@ -19,7 +19,10 @@ type Options = {
     offset?: PointLike,
     anchor?: Anchor,
     color?: string,
-    draggable?: boolean
+    draggable?: boolean,
+    rotate?: number,
```

Comment:
rotate is fine, but I wonder if `rotation` is clearer?

Reply:
Agreed, the variable has been changed to `rotation` throughout